src/_gettsim/social_insurance_contributions/eink_grenzen.py

```diff
@@ -158,10 +158,18 @@ def midijob_faktor_f_mit_minijob_steuerpauschale(
     # Therefore sum the contributions which are the same for employee and employer
     allg_sozialv_beitr = (
         sozialv_beitr_params["beitr_satz_jahresanfang"]["ges_rentenv"]
-        + sozialv_beitr_params["beitr_satz_jahresanfang"]["ges_pflegev"]["standard"]
         + sozialv_beitr_params["beitr_satz_jahresanfang"]["arbeitsl_v"]
     )
 
+    if isinstance(sozialv_beitr_params["beitr_satz_jahresanfang"]["ges_pflegev"], dict):
```

Comment:
Please get rid of this statement by making two functions out of `midijob_faktor_f_mit_minijob_steuerpauschale` according to the regime.